fe/fe-core/src/main/java/com/starrocks/qe/ConnectContext.java

```diff
@@ -118,6 +119,8 @@ public class ConnectContext {
     protected SessionVariable sessionVariable;
     // all the modified session variables, will forward to leader
     protected Map<String, SetVar> modifiedSessionVariables = new HashMap<>();
+    // user define variable in this session
+    protected HashMap<String, UserVariable> userVariables;
```

Comment:
will there be a lot of user variables?  do we support user use two variable at the same time?